packages/ERTP/src/issuer.js

```diff
@@ -169,6 +169,8 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
       );
       return payment;
     },
+    // Payments in `fromPaymentsPArray` must be distinct. Alias
+    // checking is delegated to the `reallocate` function.
```

Comment:
By my review comment, I meant that `Payments in `fromPaymentsPArray` must be distinct.` should be an API comment and `Alias checking is delegated to the `reallocate` function.` should be an implementation comment. We should establish a convention, but I would put such an implementation comment inside the body of the function.

Reply:
ok gotcha. I haven't added any JS doc style comments yet, but I will do this.